packaging/bundle-validation/validate.sh

```diff
@@ -280,6 +280,7 @@ if [ "$?" -ne 0 ]; then
 fi
 echo "::warning::validate.sh done validating spark & hadoop-mr bundle"
 
+#TODO [HUDI-8086] get this working for some of the supported spark versions
```

Comment:
Separate subtask, and we should take this on as a blocker of Hudi 1.0.

Reply:
https://issues.apache.org/jira/browse/HUDI-8295

Reply:
Could you update the ticket number in all places to use the right one instead of `HUDI-8086`?